operator/autopilot-channels/packages/configconnector/1.125.0/crds.yaml

```diff
@@ -10122,24 +10144,21 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    cnrm.cloud.google.com/version: 1.122.0
+    cnrm.cloud.google.com/version: 1.125.0
   creationTimestamp: null
   labels:
     cnrm.cloud.google.com/managed-by-kcc: "true"
     cnrm.cloud.google.com/stability-level: alpha
     cnrm.cloud.google.com/system: "true"
-    cnrm.cloud.google.com/tf2crd: "true"
   name: bigqueryanalyticshubdataexchanges.bigqueryanalyticshub.cnrm.cloud.google.com
 spec:
   group: bigqueryanalyticshub.cnrm.cloud.google.com
   names:
     categories:
     - gcp
     kind: BigQueryAnalyticsHubDataExchange
+    listKind: BigQueryAnalyticsHubDataExchangeList
     plural: bigqueryanalyticshubdataexchanges
-    shortNames:
-    - gcpbigqueryanalyticshubdataexchange
-    - gcpbigqueryanalyticshubdataexchanges
```

Comment:
Addressing in https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/3229